packages/dashboard-backend/src/routes/factoryAcceptanceRedirect.ts

```diff
@@ -37,6 +37,10 @@ export function registerFactoryAcceptanceRedirect(instance: FastifyInstance): vo
           params.append(ERROR_CODE_ATTR, querystring.unescape(query[ERROR_CODE_ATTR] as string));
         }
 
+        const url = params.get('url');
+        if (url) {
+          params.set('url', url.replaceAll(' ', '%20'));
```

Comment:
can we use `encodeURIComponent` to cover the general case? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent